### PR TITLE
JBoss Web Server example

### PR DESCRIPTION
**Pull request by rinaldodev**

### CODE REVIEW

**Review by cunningt — APPROVED** [review]
Looks good to me!   Nice job!

**Review by mcarlett — APPROVED** [review]

File: tomcat-jdbc/pom.xml [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+            <!-- Spring Boot BOM -->
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-dependencies</artifactId>
+                <version>${spring-boot-version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
```
Comment (mcarlett) [inline tomcat-jdbc/pom.xml]: I think we need to use product BOM. Thanks!

File: tomcat-jdbc/pom.xml [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+            <artifactId>spring-boot-starter-actuator</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-jdbc</artifactId>
+            <exclusions>
+                <exclusion>
+                    <!-- Spring Boot uses HikariCP DataSource by default, so we remove it -->
+                    <groupId>com.zaxxer</groupId>
+                    <artifactId>HikariCP</artifactId>
```
Comment (Croway) [inline tomcat-jdbc/pom.xml:73]: Does it mean that no pooling will be used by removing Hikari? or Spring Boot will fallback to other pooling options?

If you want to introduce pooling, you can use agroal that is supported by Red Hat and suggested in other examples like https://github.com/jboss-fuse/camel-spring-boot-examples/blob/camel-spring-boot-examples-4.4.0-branch/spring-boot-jta-jpa-autoconfigure/pom.xml#L134

**Review by Croway — APPROVED** [review]

File: tomcat-jdbc/pom.xml (hunk 2 above)
Comment (rinaldodev) [reply tomcat-jdbc/pom.xml:73]: It's using Tomcat's pooled datasource.

https://github.com/jboss-fuse/camel-spring-boot-examples/pull/145/files#diff-004fd9875f58addca5193e1af6df9a0421788fbbfb8768d1799fea8c19915d67R21

https://github.com/jboss-fuse/camel-spring-boot-examples/pull/145/files#diff-004fd9875f58addca5193e1af6df9a0421788fbbfb8768d1799fea8c19915d67R21

Comment (Croway) [reply tomcat-jdbc/pom.xml:73]: oh! okay, great then

File: tomcat-jdbc/pom.xml (hunk 1 above)
Comment (rinaldodev) [reply tomcat-jdbc/pom.xml]: Changed, thanks

File: tomcat-jdbc/pom.xml [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <!-- Camel BOM -->
+            <dependency>
+                <groupId>com.redhat.camel.springboot.platform</groupId>
+                <artifactId>camel-spring-boot-bom</artifactId>
+                <version>${camel-version}</version>
```
Comment (mcarlett) [inline tomcat-jdbc/pom.xml]: `${camel-spring-boot-version}` should be used instead of `${camel-version}`, thanks

Comment (rinaldodev) [reply tomcat-jdbc/pom.xml]: ooops, thanks!

**Pull request merged**